Caltech-IPAC/firefly
# firefly-73: Added improved cube, wavelength and spectral info support

Firefly-73: Added improved cube, wavelength and spectral info support
   - Added zero header passed and managed by client
   - converted wavelength projection type parsing to client
   - client now maintains both projections and wavelength data
   - Updated readout to show Wavelength info
   - Improved cube paging controls to page though HDU's and cubes
   - Show wave length in cube paging controls

_To test:_

https://irsawebdev9.ipac.caltech.edu/firefly-73-cube-updates/firefly/

Read in cube:

- Notice cube controls.
- Read in multi-HDU cubes, notice  cube controls
- read in cube with wavelength information should show in mouse readout and navigation controls

@lznakano  - can you pay attentions to:
 - `Wavelength.js` - This file contains the algorithms
 - `WavelengthInfo.js` - This file analyzes the headers and prepares the parameters.
